doc/source/admin/data_tables.md

```diff
@@ -0,0 +1,93 @@
+# Tool data
+
+Galaxy stores tool data in the path defined by `tool_data_path` (by default `tool-data/`).
+It's possible to to separate tool data of shed installed tools by setting (`shed_tool_data_path`).
+
+Tool data consists of:
+
+1. the actual data
+2. a tool data table
+3. tool data config files
```

Comment:
Since you can have tool data with only a loc file and no tool data table config file, I think the following is more in line with the Galaxy lingo (see e.g. https://galaxyproject.org/admin/tools/data-tables/ ).

```suggestion
2. loc files
3. tool data table config files
```

Reply:
So the term **data table** should be defined as the set of unique data defined by the loc files that are referred to by config entries using the same name?

Maybe we also want to introduce the historic perspective for tool bound loc filled? For a better understanding...

Reply:
Basically the data table is the `tool_data_table_conf.xml` file defining how to assign names to the columns in the `loc` file, which allowed Galaxy tool developers to move from using https://docs.galaxyproject.org/en/latest/dev/schema.html#from-file to https://docs.galaxyproject.org/en/latest/dev/schema.html#from-data-table .

Reply:
OK. Should we then also use the term *data table* for `tool_data_table_conf.xml` (instead of *tool data table config*)?

Reply:
I think tool data table config is fine.